src/components/menu/mobile-menu.js

```diff
@@ -188,11 +218,18 @@ export const MobileMenu = ({ children, showBrand, items }) => {
                                         </MobileSubmenuHeader>
                                         <Collapse opened={ activeSubmenus.includes(currentIndex) }>
                                             <MobileSubmenu active={ activeSubmenus.includes(currentIndex) } onClick={ handleCloseMenu }>
-                                                { item.submenu.map(subitem => <MobileMenuLink key={ subitem.path } to={ subitem.path } activeClassName="active" partiallyActive={ true }>- { subitem.text }</MobileMenuLink>) }
+                                                { item.submenu.map(subitem => isExternalLink(subitem.path) ? 
+                                                  <ExternalMobileMenuLink  key={ subitem.path } href={ subitem.path } target="_blank" rel="noreferrer" activeClassName="active" partiallyActive={ true }>- { subitem.text }</ExternalMobileMenuLink> : 
```

Comment:
You can pass an "as" prop to styled components to render a component other than what's in its definition, so i think this line could be replaced with 

```
<MobileMenuLink as="a" key={ subitem.path } href={ subitem.path } target="_blank" rel="noreferrer" activeClassName="active" partiallyActive={ true }>- { subitem.text }</MobileMenuLink> : 
```
which would use an <a> instead of Link component, as defined on line 59 above.

Then you could completely delete ExternalMobileMenuLink above. the same with non-Mobile menu links